src/styles/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  display: flex;
+  flex-direction: column;
+
+  &__header {
+    margin: 0 50px;
+  }
+
+  &__products {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+}
+
+.products__card-container {
+  padding: 50px 0 0;
+  margin: 0 auto;
+  min-width: 944px;
```

Comment:
don't hardcode width like that, the layout should be adaptive, use max-width instead, calculate it with using card-width, gap and number of cards